src/scalar.rs

```diff
@@ -23,6 +23,7 @@ use crate::constants;
 #[allow(missing_debug_implementations)]
```

Comment:
This line could have been removed in this patch.

Reply:
I'm happy to open a followup PR. Shall I?

Reply:
I went ahead and did to save time: #579. If it's just a nuisance PR due to its size, I'm happy to close it.

Reply:
Thanks man!